packages/connect/src/api/bitcoin/signtxVerify.ts

```diff
@@ -103,16 +96,17 @@ const deriveOutputScript = async (
     );
 };
 
-export const verifyTx = async (
-    getHDNode: GetHDNode,
-    inputs: PROTO.TxInputType[],
-    outputs: PROTO.TxOutputType[],
+export const decodeTx = (
```

Comment:
Is there a reason why extract `decodeTx` from `verifyTx`, when they're still always called together?